### PR TITLE
first

### PR DESCRIPTION
**Pull request by mdlevick**

### CODE REVIEW

**Comment by mdlevick** [comment]
i have finished the project with a 2

File: README.md [hunk 1]
```diff
@@ -34,30 +34,40 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
+inline is for a parent tag so that you can put them in a line. block is for children tags so that they they make a block that can be moved around.
+
 3. What are the 4 areas of the box model?
 
+margin, border, padding, and content.
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+cross axis ----.
```
Comment (srsimps19) [inline README.md]: Good job using cross-axis! due to the changing nature of where the cross-axis is!

File: index.html [hunk 2]
```diff
@@ -61,15 +70,7 @@ <h2>Moon</h2>
 
         </section>
         
-        <footer>
```
Comment (srsimps19) [inline index.html]: I think you might have forgotten somthing here!

File: index.html [hunk 3]
```diff
@@ -13,7 +13,16 @@
...
+            <img src="img/lambda-black.png" alt="lambda logo black">
+        <nav>
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+        </nav>
+       </header>
         <section class="top-content">
```
Comment (srsimps19) [inline index.html]: I think you forgot something here!

File: css/index.css [hunk 4]
```diff
@@ -90,6 +120,46 @@ p {
     border-bottom: 1px dashed black;
 }
 
+   .box:nth-child(1){
+    background: teal;
+    }
+    
+    .box:nth-child(2){
```
Comment (srsimps19) [inline css/index.css]: love the use of nth child ;)

**Pull request merged**